lib/ruled/client.lua

```diff
@@ -152,22 +152,22 @@ end
 
 --- Get list of matching rules for a client.
 -- @tparam client c The client.
--- @tparam table _rules The rules to check. List with "rule", "rule_any", "except" and
+-- @tparam table rules The rules to check. List with "rule", "rule_any", "except" and
 --   "except_any" keys.
 -- @treturn table The list of matched rules.
 -- @staticfct ruled.client.matching_rules
-function module.matching_rules(c, _rules)
-    return crules:matching_rules(c, _rules)
+function module.matchingrules(c, rules)
```

Comment:
Overzealous "find & replace" or is that change in API intended?

Reply:
Eww, can't believe we have no tests for this. Thanks for finding this.